src/Config/ConfigBuilder.php

```diff
@@ -128,6 +130,13 @@ public function setConstVariable(string $key, string $value): ConfigBuilder
         return $this;
     }
 
+    public function setImport(string $path): self
```

Comment:
Since this method doesn't overwrite `$this->imports`, but adds a new array item, I'd suggest naming this method `addImport`.
```suggestion
    public function addImport(string $path): self
```

Reply:
Was thinking about this as well, but all other methods here are prefixed `set` and use the same semantics and they come usually in a pair of two (`setConstant`/`setConstants`). But since you mention it, I will change it now for the singlular methods at least.